raiden_contracts/tests/test_user_deposit_contract.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+
+from raiden_contracts.constants import CONTRACT_USER_DEPOSIT
+
+
+@pytest.fixture
+def user_deposit_contract(
```

Comment:
We place fixtures in `tests/fixtures`, as they are generally used across multiple test files.